src/models/storage.ts

```diff
@@ -3,6 +3,79 @@
 /// <reference path="./interface.ts" />
 /// <reference path="./otp.ts" />
 
+class BrowserStorage {
+  private static getStorageLocation() {
+    if (localStorage.storageLocation !== 'sync' &&
+        localStorage.storageLocation !== 'local') {
+      return new Promise((resolve, reject) => {
+        let sync: number;
+        let local: number;
+        chrome.storage.local.get((l) => {
+          local = Object.keys(l).length;
```

Comment:
Better naming for `local`. This should be something like number of keys store in local, right?